include/seqan3/range/view/detail.hpp

```diff
@@ -239,8 +240,19 @@ class combined_adaptor :
     }
 
 public:
+    /*!\name Constructors, destructor and assignment
+     * \{
+     */
+    constexpr combined_adaptor()                                              = default; //!< Defaulted.
```

Comment:
why is the default constructor not noexcept (also for the others)?

Reply:
Because they aren't always `noexcept`. It depends on the base class.

Reply:
And `noexcept` it not like `constexpr` in that it is ok if possible and if not also ok; it hard fails if the base type's constructor is not noexcept